Qualification/input/Content/References.md

```diff
@@ -0,0 +1,51 @@
+
+## Verapamil - Digoxin DDI
+
+**Belz 1981** Belz GG, Aust PE, Munkes R. Digoxin plasma concentrations and nifedipine. Lancet. 1981 Apr 11;1(8224):844-5. doi: 10.1016/s0140-6736(81)92727-6. PMID: 6111709.
+**Belz 1983** Belz GG, Doering W, Munkes R, Matthews J. Interaction between digoxin and calcium antagonists and antiarrhythmic drugs. Clin Pharmacol Ther. 1983 Apr;33(4):410-7. doi: 10.1038/clpt.1983.55. PMID: 6831819.
+**Doering 1983** Doering W. Effect of coadministration of verapamil and quinidine on serum digoxin concentration. Eur J Clin Pharmacol. 1983;25(4):517-21. doi: 10.1007/BF00542121. PMID: 6653647.
+**Johnson 1987** Johnson BF, Wilson J, Marwaha R, Hoch K, Johnson J. The comparative effects of verapamil and a new dihydropyridine calcium channel blocker on digoxin pharmacokinetics. Clin Pharmacol Ther. 1987 Jul;42(1):66-71. doi: 10.1038/clpt.1987.109. PMID: 2954736.
+**Klein 1982** Klein HO, Lang R, Weiss E, Di Segni E, Libhaber C, Guerrero J, Kaplinsky E. The influence of verapamil on serum digoxin concentration. Circulation. 1982 May;65(5):998-1003. doi: 10.1161/01.cir.65.5.998. PMID: 7074765.
+**Pedersen 1981** Pedersen KE, Dorph-Pedersen A, Hvidt S, Klitgaard NA, Nielsen-Kudsk F. Digoxin-verapamil interaction. Clin Pharmacol Ther. 1981 Sep;30(3):311-6. doi: 10.1038/clpt.1981.165. PMID: 7273594.
+**Pedersen 1982** Pedersen KE, Dorph-Pedersen A, Hvidt S, Klitgaard NA, Pedersen KK. The long-term effect of verapamil on plasma digoxin concentration and renal digoxin clearance in healthy subjects. Eur J Clin Pharmacol. 1982;22(2):123-7. doi: 10.1007/BF00542456. PMID: 7094982.
+**Pedersen 1983** Pedersen KE, Thayssen P, Klitgaard NA, Christiansen BD, Nielsen-Kudsk F. Influence of verapamil on the inotropism and pharmacokinetics of digoxin. Eur J Clin Pharmacol. 1983;25(2):199-206. doi: 10.1007/BF00543791. PMID: 6628501.
+**Rodin 1988** Rodin SM, Johnson BF, Wilson J, Ritchie P, Johnson J. Comparative effects of verapamil and isradipine on steady-state digoxin kinetics. Clin Pharmacol Ther. 1988 Jun;43(6):668-72. doi: 10.1038/clpt.1988.93. PMID: 2967742.
+**Schwartz 1982** Schwartz JB, Keefe D, Kates RE, Kirsten E, Harrison DC. Acute and chronic pharmacodynamic interaction of verapamil and digoxin in atrial fibrillation. Circulation. 1982 Jun;65(6):1163-70. doi: 10.1161/01.cir.65.6.1163. PMID: 7074776.
+**Wiebe 2020** Wiebe ST, Giessmann T, Hohl K, Schmidt-Gerets S, Hauel E, Jambrecina A, Bader K, Ishiguro N, Taub ME, Sharma A, Ebner T, Mikus G, Fromm MF, Müller F, Stopfer P. Validation of a Drug Transporter Probe Cocktail Using the Prototypical Inhibitors Rifampin, Probenecid, Verapamil, and Cimetidine. Clin Pharmacokinet. 2020 Dec;59(12):1627-1639. doi: 10.1007/s40262-020-00907-w. PMID: 32504272; PMCID: PMC7716890.
+
```

Comment:
please reformat. at the moment it looks like this:
![{B3E81B14-361D-49C1-96B7-672B36AE7C3C}](https://github.com/user-attachments/assets/d56581d6-e8a0-4070-888e-1c2faa8bedde)